src/data/memberships.js

```diff
@@ -73,20 +80,31 @@ export default {
         muted: false,
         sharing: true,
         inMeeting: true,
+        host: true,
       },
       {
         ID: 'user2',
         orgID: 'org1',
         muted: true,
         sharing: false,
         inMeeting: false,
+        host: false,
       },
       {
         ID: 'user3',
         orgID: 'org1',
         muted: true,
         sharing: false,
         inMeeting: true,
+        host: true,
+      },
+      {
+        ID: 'user5',
```

Comment:
I think this change is related to external org users and should be part of a different PR

Reply:
You're right.
Done.